example_test.go

```diff
@@ -16,14 +16,35 @@ package e4_test
 
 import (
 	"fmt"
+	"os"
 
 	e4 "github.com/teserakt-io/e4go"
 	e4crypto "github.com/teserakt-io/e4go/crypto"
 	"golang.org/x/crypto/curve25519"
 )
 
 func ExampleNewClient_symIDAndKey() {
-	client, err := e4.NewClient(&e4.SymIDAndKey{ID: []byte("clientID"), Key: e4crypto.RandomKey()}, "./symClient.json")
+	client, err := e4.NewClient(&e4.SymIDAndKey{ID: []byte("clientID"), Key: e4crypto.RandomKey()}, e4.NewMemoryStore(nil))
+	if err != nil {
+		panic(err)
+	}
+
+	protectedMessage, err := client.ProtectMessage([]byte("very secret message"), "topic/name")
+	if err != nil {
+		panic(err)
+	}
+
```

Comment:
Please remove this newline, it is logically attached to the prior block as an alternative, so
```go
	protectedMessage, err := client.ProtectMessage([]byte("very secret message"), "topic/name")
	if err != nil {
		panic(err)
	}
        fmt.Printf("Protected message: %v", protectedMessage)
```